### PR TITLE
fix: the url key in seo component

### PR DESCRIPTION
**Pull request by zielu92**
Now it would display the slug + url key. Previously it was slug + meta title

## Description
Adding the CMS page would skip providing full URL in preview and provide slug + meta title

## How To Test This?
Try to add the new CMS page

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/views/components/seo/index.blade.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ class="text-[#161B9D] dark:text-white"
             <!-- SEO Meta Title -->
             <p 
                 class="text-[#135F29]"
-                v-text="'{{ URL::to('/') }}/' + (slug ? slug + '/' : '') + (metaTitle ? metaTitle.toLowerCase().replace(/\s+/g, '-') : '')"
+                v-text="'{{ URL::to('/') }}/' + (slug ? slug + '/' : '') + (urlKey ? urlKey().replace(/\s+/g, '-') : '')"
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/components/seo/index.blade.php:19]: Remove ()  from urlKey() it is a data property.

File: packages/Webkul/Admin/src/Resources/views/components/seo/index.blade.php [hunk 2]
```diff
@@ -38,7 +38,7 @@ class="text-gray-600 dark:text-gray-300"
             data() {
                 return {
                     metaTitle: this.$parent.getValues()['meta_title'],
-
+                    urlKey: this.$parent.getValues()['url_key'],
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/components/seo/index.blade.php:41]: Manage proper spacing.

**Comment by zielu92** [comment]
fixed issues

**Comment by devansh-webkul** [comment]
@zielu92,

In the starting phase url key is null, and the focus URL key gets attached. Please find the video,


https://github.com/bagisto/bagisto/assets/68321766/33b5e777-6d9a-437e-8137-2c7827c4754c

**Comment by devansh-webkul** [comment]
Also this component used in more places, check the channel edit page also.

**Comment by devansh-webkul** [comment]
Hello @zielu92,

Any update?

**Comment by zielu92** [comment]
I will update within this week

**Comment by devansh-webkul** [comment]
> I will update you within this week. BTW the video does not work.

Use chrome.

**Comment by devansh-webkul** [comment]
We will do it internally, after discussion with the team.

**Pull request closed without merging**